app/src/main/res/values/strings.xml

```diff
@@ -1,6 +1,6 @@
 <resources>
     <!--General Strings-->
-    <string name="app_name" translatable="false">Susi</string>
```

Comment:
Any particular reason to remove `translatable="false` from this line?

Reply:
we can't keep two variables of the same name while giving the `translatable = "false"`. Either we remove one of the string or remove `translatable = "false"`.

Reply:
Remove it from other translations. Keep one in default and assign `translatable = "false"`

Reply:
I tried that it works with other strings but it is necessary to keep the `app_name`  variable otherwise app crashes with SiGSEGV error.

Reply:
Ohh. Didn't know that. Fine then.